prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -91,61 +86,35 @@ def build_agent(
 
     agent.max_tries = 3
 
-    # Restore the prompt from a historical session, replacing the editable part with it.
-    if prompt_handler:
-        if historical_prompt := prompt_handler.fetch_latest_prompt():
-            system_prompt = build_full_system_prompt(historical_prompt.prompt)
-
-    agent.system_prompt = system_prompt.format(engine_help=agent.engine.help)
+    agent.system_prompt = unformatted_system_prompt.format(
+        engine_help=agent.engine.help
+    )
     if bootstrap:
         agent.bootstrap = [bootstrap]
     return agent
 
 
-def main(
```

Comment:
Remove, as we have `Deployed...Agent.run` that we can call from prediction_market_agent/run_agent.py